### ISSUE TITLE
When trying to select an existing Project, User gets prompted to create a New project instead

### ISSUE THREAD

**Issue by zdericnina** [toggl-open-source/toggldesktop#3469]
### 💻 Environment

Platform:  macOS
OS Version:  macOS 10.14.6
Toggl Version:  7.4.1023.

### 🐞 Actual behavior

When filling in the details for their ongoing time entry, when trying to select the Project, user gets prompted to create a new project rather than select it from the list

### 💯 Expected behavior

User should be able to easily select the Project they want to

### 🔨 Steps to reproduce

I haven't been able to replicate, clearing local data manually didn't help with resolving the issue.

### 📦 Additional info

[Screencast/User report](https://app.intercom.io/a/apps/ayixs927/inbox/inbox/2759100/conversations/24187998764)

**Comment by nidarasheed** [comment]
[Case reference](https://app.intercom.io/a/apps/ayixs927/inbox/inbox/534741/conversations/24860002265)

**Comment by NghiaTranUIT** [comment]
Close because this ticket is resolved in > 7.4.1036.

1. First User reference
- Link: https://app.intercom.io/a/apps/ayixs927/inbox/inbox/2759100/conversations/24187998764
-> She confirmed that it fixed in a BETA 7.4.1036
```
I did, however, notice another bug where the time entry details box would disappear randomly... I've attached a recording which shows this. Hopefully that helps
```
=> Fix in different ticket, but I couldn't find it for now.

2. The second user case is still using old version 7.4.1031 -> Update to the latest version would fix it 👍
